utils/wallet/provider/uphold/uphold.go

```diff
@@ -424,6 +424,17 @@ type transactionRequest struct {
 	Message      string       `json:"message,omitempty"`
 }
 
+type denominationRecode struct {
```

Comment:
can you add a quick comment for why we added this? otherwise this looks good to me

Reply:
can do

Reply:
done in d840a24